.i18nrc.json

```diff
@@ -46,9 +46,8 @@
     "xpack.watcher": "x-pack/plugins/watcher"
   },
   "exclude": [
-    "src/core/public/fatal_errors/get_error_info.ts",
-    "src/legacy/ui/ui_render/bootstrap/app_bootstrap.js",
-    "src/legacy/ui/ui_render/ui_render_mixin.js",
+    "src/ui/ui_render/bootstrap/app_bootstrap.js",
+    "src/ui/ui_render/ui_render_mixin.js",
```

Comment:
Hi @jinmu03, I see that `ui` folder is still in `legacy` folder. So the right path to `ui_render` is `src/legacy/ui/ui_render`. 
```js
"src/legacy/ui/ui_render/bootstrap/app_bootstrap.js",
"src/legacy/ui/ui_render/ui_render_mixin.js",
```

That should fix the CI.